homeassistant/components/ambient_station/sensor.py

```diff
@@ -296,6 +298,12 @@
         native_unit_of_measurement="strikes",
         state_class=SensorStateClass.TOTAL,
     ),
+    SensorEntityDescription(
```

Comment:
Device class should be set to `distance` I believe

Reply:
CI is also failing and needs to be addressed

Reply:
fixed device class